docs/administrator-documentation/moderne-platform/references/reporting.md

```diff
@@ -23,7 +26,11 @@ Audit logs are accessible from `https://<TENANT>.moderne.io/admin/audit-logs`, a
 
 To download audit logs, use the "Export to CEF" button: ![](./assets/audit-log-download.png)
 
-## Other reports (recipe runs and commits)
+## Usages by organization
+
+The usages report is accessible from `https://<TENANT>.moderne.io/devcenter/{organization}`
+
+## Other reports (recipe runs, commits and usages)
 
 To access non-audit-log reports, navigate to `https://<TENANT>.moderne.io/admin/reports`.
 
```

Comment:
The figure isn't showing up when in the `display the rich diff`, if this expected? This wasn't change so I was a bit surprised but I would like confirm I didn't break the link

Reply:
The rich diff is not an accurate view of what the page will look like. If you want to test what it will all look like, run `yarn start` from your command line locally. That will start up a local Docusaurus server and you can view the site there.

Reply:
thanks for the clarification